packages/compiler/src/internals/init.ts

```diff
@@ -0,0 +1,2 @@
+export { NodeSystemHost } from "../core/node-system-host.js";
+export { scaffoldNewProject } from "../init/scaffold.js";
```

Comment:
i think you also need to export validateTemplate method or template schema for vscode to do all the template validation.